test_receiver.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
```

Comment:
@TheSavior :( sorry about my laziness.
@ProjectBarks may you be my reinforcement?